### PR TITLE
Update url for ABP Iceland

### PR DESCRIPTION
**Pull request by ryanbr**
Update URL to match the newly hosted list on https://github.com/brave/adblock-lists

### CODE REVIEW

File: components/third_party/adblock/lists/is_abp/README.chromium [hunk 1]
```diff
@@ -1,4 +1,4 @@
 Name: Icelandic ABP List
-URL: https://adblock.gardar.net/
+URL: https://github.com/brave/adblock-lists
```
Comment (fmarier) [inline components/third_party/adblock/lists/is_abp/README.chromium:2]:
```suggestion
URL: https://github.com/brave/adblock-lists/blob/master/custom/is.txt
```

**Review by fmarier — CHANGES_REQUESTED** [review]

**Review by fmarier — APPROVED** [review]

**Comment by ryanbr** [comment]
Not sure whats hold this up @fmarier

**Comment by fmarier** [comment]
@ryanbr Looks like CI is stuck. Try squashing your two commits and then doing a `git push --force`, it will trigger a new CI run.

**Comment by ryanbr** [comment]
Dropping this its pulled in other commits for some reason

**Pull request closed without merging**